src/mono/mono/metadata/cominterop.c

```diff
@@ -4155,6 +4155,7 @@ ves_icall_System_Runtime_InteropServices_Marshal_QueryInterfaceInternal (MonoIUn
 #endif /* HOST_WIN32 */
 #endif /* DISABLE_COM */
 
+#ifndef ENABLE_NETCORE
 MonoStringHandle
```

Comment:
Is this related ?

Reply:
Yes, it's a warning that slipped in because this wasn't actually working.